application-analytics-api/src/main/java/com/xwiki/analytics/configuration/AnalyticsConfiguration.java

```diff
@@ -46,4 +46,9 @@ public interface AnalyticsConfiguration
      * @return the authentication token that permits to access the statistics
      */
     String getAuthenticationToken();
+
+    /**
+     * @return the tracking code for Matomo.
```

Comment:
Isn't "Matomo" just an implementation detail? I suppose the need for a tracking code is generic, for any external analytics service used.